nni/compression/pytorch/speedup/jit_translate.py

```diff
@@ -125,6 +125,27 @@ def add_python(node, speedup):
         return new_add
 
 
+def sub_python(node, speedup):
+    c_node = node.key_node
+    inputs = list(c_node.inputs())
+    constant = None
+    for i in range(2):
+        input_i = inputs[i]
+        debug_name = input_i.debugName()
+        if debug_name not in speedup.internal_result:
+            # this input is a constant value
+            # TODO: what if this input is a constant tensor
+
+            if input_i.toIValue() is not None:
+                constant = parse_constant(input_i, speedup)
+                break
```

Comment:
I am a little confused about this part. what is constant here?

Reply:
constant here means maybe in your model, you have such operation:
```
output = torch.sub(x, 10)
```
here, `10` is constant.